src/services/pcn-iptables/src/datapaths/Iptables_Horus_dp.c

```diff
@@ -133,18 +132,18 @@ static int handle_rx(struct CTXTYPE *ctx, struct pkt_metadata *md) {
   struct horusValue *value;
   value = horusTable.lookup(&key);
 
-  pcn_log(ctx, LOG_DEBUG, "HORUS key: %x. pkt: %x", key.srcIp, pkt->srcIp);
+  pcn_log(ctx, LOG_DEBUG, "Horus srcIp: %I dstIp: %I", pkt->srcIp, pkt->dstIp);
   if (value == NULL) {
     // Miss, goto pipleline
     goto PIPELINE;
   } else {
     // Independently from the final action (ACCEPT or DROP)
     // I have to update the counters
-    if (value->ruleID <= 1024) {
+    if (value->ruleID <= _MAX_RULE_SIZE_FOR_HORUS) {
```

Comment:
I think here the test has to be `<=`.
The range of ruleID is `[0, _MAX_RULE_SIZE_FOR_HORUS - 1]`

Reply:
The test has to be `<`

Reply:
I meant `<`, this is very easy to get confused :D